network-boot-lab.md

```diff
@@ -0,0 +1,282 @@
+# Launch EOSIO network
+
+## Workshop instructions
+
+Complete the instructions described in this lab.
+
+The student must send:
+1. A `zip file` containing the final folder after completing this tutorial and just removing the `data` folder.
+1. A screenshot of the terminal with nodeos running after the lab is completed.
+
+### Help material
+
+Most of the resource here shown are taken from the `EOSIO Private Network Installation` tutorial, to follow the complete example, please refer to [this link](https://guide.eoscostarica.io/docs/tutorials/private-network-installation/#block-producer-nodes-configuration-and-execution).
+
+## Goal
+
+Launch an EOSIO Network locally with the needed nodeos plugins and install the system contracts that teach new rules to the network about how to behave.
+
+## Software Requirements
+
+At the time of configuring a private EOSIO network there are some software installation requirements:
+1. EOSIO precompiled binaries installation
+1. EOSIO.CDT binaries installation
+1. cmake
+1. build-essential
+
+*Also, we are using Ubuntu 18.04 LTS, but if you want to use macOS or other supported OS, take a look at [EOSIO releases](https://github.com/EOSIO/eos/releases)*
+
+### EOSIO Precompiled Binaries Installation
+Execute the following commands to install EOSIO precompiled binaries:
```

Comment:
this works on debian linux platforms , what if the user is on a mac or windows or centos linux?

Reply:
Solved